### PR TITLE
Initialize brancgh. Add problem and solution to reverse-a-queue-chall…

### PR DESCRIPTION
**Pull request by anthonykyi**
…enge.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

**Comment by RandellDawson** [comment]
Please review the instructions for [how to create a new challenge](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-curriculum-challenges.md).  Was this discussed in an issue prior to you creating this PR?

**Comment by anthonykyi** [comment]
@RandellDawson This challenge was not discussed prior to creating this PR.
I was testing it to see if it would pass the test.  I found out the problems I had with it and I believe I resolved it.  If it needs anymore changing, please let me know.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
In addition to the suggestion I made, I suggest adding other tests where the items are strings and are not in numeric or alphabetic order, or someone could just simply use `sort` method on the array.

**Comment by anthonykyi** [comment]
@RandellDawson I added 1 additional test case as you suggested!
Tell me how it looks!

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/reverse-a-queue.english.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+<section id='description'>
+Since you know what a queue is, this challenge will go over a queue again.
+
+</section>
+
+## Instructions
+<section id='instructions'>
+In this challenge, you will create a method that takes in a queue and reverses it.
+For example, if I put in a queue that with the integers 
+1,2,3,4,5, the output will then be 5,4,3,2,1.s
```
[the remark this answers is not in the document]
Comment (anthonykyi) [reply curriculum/challenges/english/08-coding-interview-prep/data-structures/reverse-a-queue.english.md]: @RandellDawson I just changed the code according to your suggested change!

[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/data-structures/reverse-a-queue.english.md]: You did not add additional tests like I specified above.

[the remark this answers is not in the document]
Comment (anthonykyi) [reply curriculum/challenges/english/08-coding-interview-prep/data-structures/reverse-a-queue.english.md]: I added the following test to include strings..
And I will get to enqueue and dequeue right away.

- text: Your stringed <code>data</code> should be a reverse of itself.
    testString: assert((function(){var testQueueString = new Queue(['hello', 'world']); testQueueString.reverse(); return JSON.stringify(testQueueString.data) === JSON.stringify(['world', 'hello'])}()), 'Your stringed <code>data</code> should be a reverse of itself' );

[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/data-structures/reverse-a-queue.english.md]: Your tests would use the enqueue method to add the multiple items to the queue.  I recommend having at least 4 items for the tests which adds the string items.

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
The challenge would need to be added to `meta.json` before we can proceed with this

**Comment by RandellDawson** [comment]
Hi @anthonykyi 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!

**Pull request closed without merging**